lib/ansible/errors/__init__.py

```diff
@@ -220,7 +220,24 @@ class AnsibleUndefinedVariable(AnsibleRuntimeError):
 
 class AnsibleFileNotFound(AnsibleRuntimeError):
     ''' a file missing failure '''
-    pass
+
+    def __init__(self, message="", obj=None, show_content=True, suppress_extended_error=False, orig_exc=None, paths=None, file_name=None):
```

Comment:
maybe s/paths/searched_paths to distinguish from 'file_name' a little more? Though the message formatting can tweak that however for using facing messages so doesnt matter a ton.